master doesn't compile

```gradle clean buildPlugin``` produces the following compilation error:
```
/Users/andredietisheim/Documents/jboss-workspaces/intellij-tekton/src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/TknCli.java:492: error: cannot find symbol
            ExecHelper.executeWithUI(envVars, outputToEditor(fileName),command, "pipelinerun", "logs", pipelineRun, "-f", "-n", namespace);
                      ^
  symbol:   method executeWithUI(Map<String,String>,Consumer<String>,String,String,String,String,String,String,String)
  location: class ExecHelper
```

--- a/build.gradle
+++ b/build.gradle
@@ -29,7 +29,7 @@ intellij {
 dependencies {
     compile 'io.fabric8:openshift-client:5.0.0'
     compile 'io.fabric8:tekton-client:5.0.0'
-    compile 'com.redhat.devtools.intellij:intellij-common:0.0.1-SNAPSHOT'
+    compile 'com.redhat.devtools.intellij:intellij-common:0.0.1'
     testCompile 'org.mockito:mockito-inline:3.4.6'
     compile 'com.github.graphstream:gs-core:dev-SNAPSHOT'
     compile 'com.github.graphstream:gs-ui-swing:dev-SNAPSHOT'

